src/app-layer-ftp.c

```diff
@@ -552,6 +597,29 @@ static int FTPParseResponse(Flow *f, void *ftp_state, AppLayerParserState *pstat
         }
     }
 
+    if (state->command == FTP_COMMAND_EPRT) {
+        uint16_t dyn_port;
+        dyn_port = FTPGetV6PortNumber(state->port_line, state->port_line_len);
```

Comment:
We don't use Rust implementation here with current code.

Reply:
Correct -- that's because the `state->port_line` will be something like this: `"EPRT |2|2601:191:8500:2e00:7c0f:78e0:dc5b:f7c1|34325|"` (and similar when using IPv4).

The current Rust functions called from the passive handlers handle an input line like `229 Entering Extended Passive Mode (|||48758|)`